prisma/seed.js

```diff
@@ -0,0 +1,203 @@
+const { PrismaClient } = require('@prisma/client');
+const prisma = new PrismaClient();
+const { createFakeAdmin, createFakeCompetition, createFakeCompetitors, createFakeSeasons, createFakeTeam, createFakeRound, createFakePlacement, createFakePositionMapping } = require('../src/server/utils/faker');
```

Comment:
Prettier should have split this line up? That is one gigantic line

Reply:
ok I fix